### PR TITLE
Enable Admin Bundle in website context

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no 
| New feature? | yes
| BC breaks? | no 
| Deprecations? | no
| Fixed tickets | fixes # 
| Related issues/PRs | part of #5902
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Enable Admin Bundle in website context.

#### Why?

The SuluAdminBundle contains services like list and form metadata which also make sense to be used on the website context, also in future we should make sure that sulu works with a single kernel so step by step we should make sure that there is no longer any context based service.

#### To Do

- [ ] Create a documentation PR
- [ ] Add breaking changes to UPGRADE.md

### CODE REVIEW

File: src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php [hunk 1]
```diff
@@ -103,7 +104,6 @@ public function registerBundles()
 
         if (self::CONTEXT_ADMIN === $this->getContext()) {
             $bundles[] = new \Symfony\Bundle\SecurityBundle\SecurityBundle();
-            $bundles[] = new \Sulu\Bundle\AdminBundle\SuluAdminBundle();
```
Comment (niklasnatter) [inline src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php:106]: shouldnt we do this for the other bundles in this block too? 🤔

Comment (alexander-schranz) [reply src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php:106]: its a step by step, take some work todo for all.

**Comment by niklasnatter** [comment]
Related to https://github.com/sulu/skeleton/pull/110

File: src/Sulu/Component/HttpKernel/SuluKernel.php [hunk 2]
```diff
@@ -82,6 +82,14 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $container->addResource(new FileResource($this->getProjectDir() . '/config/bundles.php'));
         $confDir = $this->getProjectDir() . '/config';
 
+        // this is a bc layer that sulu_admin.yaml is only loaded when bundle is registered
+        // this can be removed when there is no longer context based configuration
+        $bundles = $container->getParameter('kernel.bundles');
+        if (isset($bundles['SuluAdminBundle'])) {
```
Comment (alexander-schranz) [inline src/Sulu/Component/HttpKernel/SuluKernel.php:89]: @nnatter this fixes your comment here: https://github.com/sulu/skeleton/pull/110#issuecomment-822359021

**Pull request merged**